editor/draw.js

```diff
@@ -31,6 +33,21 @@ var RandomizeModes = {
 var randomize_ids = RandomizeModes.LET_DOCUMENT_DECIDE;
 
 /**
+ * Add class 'layer' to the element
+ *
+ * Parameters:
+ * @param {SVGGElement} elem - The SVG element to update
+ */
+function addLayerClass(elem) {
+	var classes = elem.getAttribute('class')
+	if (classes === null || classes === undefined || classes.length === 0) {
+		elem.setAttribute('class', LAYER_CLASS);
+	} else if (! LAYER_CLASS_REGEX.test(classes)) {
+		elem.setAttribute('class', classes + ' ' + LAYER_CLASS);
+	}
```

Comment:
My god!  I thought the days of worrying about IE support were long gone!  At least Edge supports it - sadly IE11 has a much higher marketshare than Edge still... *sniff*